### DISCUSSION TITLE
Mesh alignment problem

### DISCUSSION THREAD

**Discussion by Purnasundar** [firemodels/fds#13711] [Q&A]
I have an issue regarding meshes with velocity and pressure along with radiation losses error. What to do?
[8.txt](https://github.com/user-attachments/files/17651962/8.txt)
![Mesh Error](https://github.com/user-attachments/assets/3ca479f0-632e-44ae-8c4d-860eddd8dbea)

**Comment by mcgratta** [comment]
I do not see a problem here.

**Comment by Purnasundar** [comment]
The code had run for 561 s and then is showing error. Further it doesn't work. Can you help me out please?

**Comment by mcgratta** [comment]
What is the error?

**Comment by Purnasundar** [comment]
pressure and velocity errors

**Comment by Purnasundar** [comment]
have you checked the coding file? do you find any errors?

**Comment by mcgratta** [comment]
What exactly is the error message?

**Comment by rmcdermo** [comment]
My guess is that the user is looking at the .out file and seeing the errors reported there.

@Purnasundar All CFD codes have error, these need to be judged relative to a tolerance you as a user need to set.  Most likely, these errors are benign.

**Reply by Purnasundar** [reply]
Okay . But such errors when pop up the screen what can be done? should I set these errors by limts?

**Comment by mcgratta** [comment]
I am running the case. It has reached 514 s and is still running.

**Comment by Purnasundar** [comment]
I don't know why this is happening on my laptop. Can we speed up the simulation? As I have been performing this simulation for the longest time, around a year, the simulations are still not over. I am feeling helpless at this point. can you please help me through?

**Comment by Purnasundar** [comment]
I have fine mesh simulations which are dead slow. But I need the results. Can you help me by giving an access to something like cloud server?

**Comment by gforney** [comment]
google fds cloud server . you'll get some hits

**Comment by Purnasundar** [comment]
Okay, I am going to try on the server for quicker results. Thanks a lot.

**Comment by mcgratta** [comment]
The case ran successfully with the latest source code. If you are not using the latest official release, upgrade. If you are using the latest release, try the latest nightly build.
